configs/TUNERCF435/config.h

```diff
@@ -0,0 +1,106 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU        AT32F435G
+
+#define BOARD_NAME           TUNERCF435
+#define MANUFACTURER_ID      TURC
+
+#define USE_ACC
+#define USE_GYRO
+
+#define USE_ACCGYRO_BMI270
+
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO_SPI_ICM42688P
+
+#define USE_BARO
+#define USE_BARO_DPS310
+
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+
+#define USE_MAX7456
+
+#define USE_BEEPER
+#define BEEPER_PIN              PB2
+
+#define MOTOR1_PIN              PB9
+#define MOTOR2_PIN              PB8
+#define MOTOR3_PIN              PB7
+#define MOTOR4_PIN              PB6
+
+#define LED_STRIP_PIN           PA8
+
+#define UART1_TX_PIN            PA9
+#define UART2_TX_PIN            PA2
+#define UART3_TX_PIN            PB11
+#define UART4_TX_PIN            PA0
+#define UART1_RX_PIN            PA10
+#define UART2_RX_PIN            PA3
+#define UART3_RX_PIN            PB10
+#define UART4_RX_PIN            PA1
+
+#define INVERTER_PIN_UART4		PC15
```

Comment:
Please use spaces instead of tabs

Reply:
The config file is modified.
Please check it out.
Thank you.